src/components/AppLayout/Header/components/Layout.tsx

```diff
@@ -57,6 +57,8 @@ const styles = () => ({
   },
   popper: {
     zIndex: 1301,
+    borderRadius: '8px',
+    boxShadow: 'rgb(40 54 61 / 18%) 1px 2px 10px 0px',
```

Comment:
This is not working so well 
<img width="319" alt="Screenshot 2022-05-25 at 13 49 55" src="https://user-images.githubusercontent.com/32431609/170255961-de9dac4f-628f-4620-907a-1f84dcb942c8.png">